prescient/simulator/master_options.py

```diff
@@ -945,40 +421,30 @@ def _construct_inner_options_parser():
                                          action='store_true',
                                          dest='output_ruc_dispatches',
```

Comment:
Never used, consider removing it.

Reply:
I'm a bit surprised this option got disabled...though I'm sure approved a PR that did so. I wouldn't mind seeing this and the `output_sced_solutions` option restored at some point -- a not unheard of way to debug Prescient is to enable these options and dump the Prescient output to a file.

Reply:
Removed.